packages/sharing-editor/bin/gen-sample-app-manifests-json.ts

```diff
@@ -3,25 +3,27 @@
 import path from "path";
 import fsPromises from "fs/promises";
 import fsExtra from "fs-extra";
-import { Record, String, Array, Static } from "runtypes";
+import * as s from "superstruct";
 import { parseRequirementsTxt } from "@stlite/common";
 
 const rootDirPath = path.resolve(__dirname, "..");
 const sampleAppRootDirPath = path.join(rootDirPath, "./public/samples");
 const srcDirPath = path.join(rootDirPath, "./src");
 const jsonOutputPath = path.join(srcDirPath, "./sample-app-manifests.json");
 
-const SampleAppRawManifest = Record({
-  title: String,
-  entrypoint: String,
+const SampleAppRawManifestStruct = s.object({
+  title: s.string(),
+  entrypoint: s.string(),
 });
-type SampleAppRawManifest = Static<typeof SampleAppRawManifest>;
-const SampleAppManifest = SampleAppRawManifest.extend({
-  files: Array(String),
-  requirements: Array(String),
-  basePath: String,
+type SampleAppRawManifest = s.Infer<typeof SampleAppRawManifestStruct>;
+const SampleAppManifestStruct = s.object({
```

Comment:
Use `s.assign` to merge `SampleAppRawManifestStruct`'s fields instead of simply adding `title` and `entrypoint`.